en/orm/retrieving-data-and-resultsets.rst

```diff
@@ -844,45 +844,6 @@ result set.
 Changing Fetching Strategies
 ============================
 
-As you may know already, ``belongsTo`` and ``hasOne`` associations are loaded
-using a ``JOIN`` in the main finder query. While this improves query and
-fetching speed and allows for creating more expressive conditions when
-retrieving data, this may be a problem when you want to apply certain clauses to
-the finder query for the association, such as ``order()`` or ``limit()``.
-
-For example, if you wanted to get the first comment of an article as an
```

Comment:
> if you wanted to get the first comment of an article

What alternative is there to fetch such an article? Should we mention maybe the alternative way to do so?

Reply:
The cleanest way is create a new relation
https://gist.github.com/ADmad/0a451d837f019edf3447

Reply:
Should we document this instead?

Reply:
probably